python/sglang/srt/managers/scheduler_metrics_mixin.py

```diff
@@ -90,7 +90,7 @@ def log_prefill_stats(
 
         num_new_seq = len(can_run_list)
         f = (
-            f"Prefill batch. "
+            f"Prefill batch. iter: {self.forward_ct + 1}, "
```

Comment:
```suggestion
            f"Prefill batch [{self.forward_ct + 1}], "
```

Try to make it more concise. Do the same to decode.

Reply:
Done